From 454702a3fa207ae5553627a60cfd36b36e2b344c Mon Sep 17 00:00:00 2001
From: Butui Hu <hot123tea123@gmail.com>
Date: Wed, 21 Feb 2024 10:56:46 +0800
Subject: [PATCH 4/5] use external opencv headers

Arch packages OpenCV 5 headers under /usr/include/opencv5 and renamed
libopencv_calib3d/libopencv_features2d to libopencv_calib/libopencv_features.

---
 third_party/opencv_linux.BUILD | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/third_party/opencv_linux.BUILD b/third_party/opencv_linux.BUILD
--- a/third_party/opencv_linux.BUILD
+++ b/third_party/opencv_linux.BUILD
@@ -18,19 +18,19 @@
         #"include/aarch64-linux-gnu/opencv4/opencv2/cvconfig.h",
         #"include/arm-linux-gnueabihf/opencv4/opencv2/cvconfig.h",
         #"include/x86_64-linux-gnu/opencv4/opencv2/cvconfig.h",
-        #"include/opencv4/opencv2/**/*.h*",
+        "include/opencv5/opencv2/**/*.h*",
     ]),
     includes = [
         # For OpenCV 4.x
         #"include/aarch64-linux-gnu/opencv4/",
         #"include/arm-linux-gnueabihf/opencv4/",
         #"include/x86_64-linux-gnu/opencv4/",
-        #"include/opencv4/",
+        "include/opencv5/",
     ],
     linkopts = [
         "-l:libopencv_core.so",
-        "-l:libopencv_calib3d.so",
-        "-l:libopencv_features2d.so",
+        "-l:libopencv_calib.so",
+        "-l:libopencv_features.so",
         "-l:libopencv_highgui.so",
         "-l:libopencv_imgcodecs.so",
         "-l:libopencv_imgproc.so",
-- 
2.43.2
